internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -804,22 +804,34 @@ func (r *Reconciler) reconcileReferencedTemplate(ctx context.Context, in reconci
 func createErrorWithoutObjectName(err error, obj client.Object) error {
```

Comment:
Q: should we adapt unit tests to ensure those use case are covered (and future regressions)